universe/com.goal_smart/device.js

```diff
@@ -0,0 +1,275 @@
+    this.name = 'GoalSmart';
+    this.description = 'Keep up with the latest in soccer';
+  }
+
+  // getting league standings, given league id
+  get_standings({ league_id }) {
+    let month = new Date().getMonth();
+    let date = new Date().getFullYear() - 1;
+    if (month > 7) {
+      date = new Date().getFullYear();
+    }
+    return Tp.Helpers.Http.get('https://api-football-v1.p.rapidapi.com/v3/standings?season=' + date + '&league=' + league_id, {
+      extraHeaders: {
+        'x-rapidapi-key': this.constructor.metadata.auth.api_key,
+        'x-rapidapi-host': 'api-football-v1.p.rapidapi.com',
+      },
+      accept: 'application/json'
+    }).then((tempResponse) => {
+      const a = JSON.parse(tempResponse);
+      const b = a.response[0].league.standings[0];
+      return b.map((obj) => {
+        const r = obj.rank;
+        const n = obj.team.name;
+        const t_id = obj.team.id;
+        const f = obj.form;
+        let fArray = [];
+        let length = f.length;
+        for (let i = 0; i < length; i++) {
+          fArray.push(f[i]);
+        }
+        const p = obj.points;
+        return ({
+          team: new Tp.Value.Entity(String(t_id), String(n)),
+          rank: r,
+          form: fArray,
+          points: p
+        });
+      });
+
+    });
+  }
+  // getting the upcoming fixtures of a team, given team id
+  get_teamFixtures({ team_id }) {
+    return Tp.Helpers.Http.get('https://api-football-v1.p.rapidapi.com/v3/fixtures?team=' + team_id + '&next=5', {
+      extraHeaders: {
+        'x-rapidapi-key': this.constructor.metadata.auth.api_key,
+        'x-rapidapi-host': 'api-football-v1.p.rapidapi.com',
+        useQueryString: true
+      },
+      accept: 'application/json'
+    }).then((tempResponse) => {
+      const a = JSON.parse(tempResponse);
+      const b = a.response;
+      return b.map((obj) => {
+        const l = obj.league.name;
+        const l_id = obj.league.id;
+        const n1 = obj.teams.home.name;
+        const t1_id = obj.teams.home.id;
+        const n2 = obj.teams.away.name;
+        const t2_id = obj.teams.home.id;
+        const v = obj.fixture.venue.name;
+        return ({
+          team: new Tp.Value.Entity(String(l_id), String(l)),
+          team1: new Tp.Value.Entity(String(t1_id), String(n1)),
+          team2: new Tp.Value.Entity(String(t2_id), String(n2)),
+          venue: v
+        });
+
+      });
+
+    });
+
+  }
+  // getting the upcoming fixtures of a team, given team id
+  get_previousFixtures({ team_id }) {
+    return Tp.Helpers.Http.get('https://api-football-v1.p.rapidapi.com/v3/teams?id=' + team_id, {
+      extraHeaders: {
+        'x-rapidapi-key': this.constructor.metadata.auth.api_key,
+        'x-rapidapi-host': 'api-football-v1.p.rapidapi.com',
+        useQueryString: true
+      },
+      accept: 'application/json'
+
+    }).then((tempResponse1) => {
+      const a1 = JSON.parse(tempResponse1);
+      const b1 = a1.response;
```

Comment:
Youre not using `a1` or `b1` entirely, and LGTM correctly complains. Why are you making an API call that you're not using? Something is off here.